projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.ts

```diff
@@ -56,6 +59,15 @@ import { ActionParameterConstraint, ConstraintValue } from '../../model/action-p
   ]
 })
 export class RuleActionUiComponent implements ControlValueAccessor, OnInit, OnDestroy {
+  private _nodeId = '';
+  @Input()
+  get nodeId(): string {
+    return this._nodeId;
+  }
+  set nodeId(value) {
+    this._nodeId = value;
+  }
```

Comment:
There's no point to doing this as opposed to just a normal property. Using a setter is useful if you have something extra to do to a value every time it's passed into the component.
```suggestion
  @Input()
  nodeId = '';
```